[WebProfilerBundle] Tweaked the Cache profiler panel

javiereguiluz opened the pull request:
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Main changes:

---

1) When there are no calls in a cache pool, now we display less information:

| Before / After
| ---
| ![before-empty-pool](https://cloud.githubusercontent.com/assets/73419/24652670/fdaa22f6-1932-11e7-8075-1fbcb8f333e2.png)
| ![after-empty-pool](https://cloud.githubusercontent.com/assets/73419/24652672/ffbe3816-1932-11e7-920f-5e0fdd880953.png)

---

2) The "call list" is now displayed in a more compact way:

![after-compact-call-list](https://cloud.githubusercontent.com/assets/73419/24652706/1dfed9fc-1933-11e7-9b56-b453bf87a2ea.png)

Comparison:

| Before | After
| --- | ---
| ![before-call-list](https://cloud.githubusercontent.com/assets/73419/24652721/2935a3d2-1933-11e7-96a5-3bffb93807a1.png) | ![after-call-list](https://cloud.githubusercontent.com/assets/73419/24652723/2aa77466-1933-11e7-9e41-70656b42f086.png)

---

@ro0NL I know that you had a discussion about showing 0% or n/a for the ratio when there are no calls. You settled for `n/a` ... but as you can see, using `0%` allows to simplify templates ... and is still technically correct. I hope you understand it.

javiereguiluz committed:
[WebProfilerBundle] Tweaked the Cache profiler panel

ro0NL commented:
Looks good :+1: about the table list;

- i would make the time column the most right one
- are complex values by default collapsed?
- `Arguments` should be `Argument`
- i would add a dot after `No calls were made..` (most empty blocks do this i noticed)
- aligning it horizontally is probably a good idea; just make sure argument/result have the most space

javiereguiluz committed:
Minor changes

javiereguiluz commented:
> I would make the time column the most right one

I prefer the leftmost column because that's where we display the time in other panels (e.g. Doctrine)

> are complex values by default collapsed?

Semi-collapsed: depth = 1

> Arguments should be Argument

Fixed!

> I would add a dot after No calls were made.. (most empty blocks do this i noticed)

Fixed!

> aligning it horizontally is probably a good idea; just make sure argument/result have the most space

We're lucky because this should work OK in most of cases. The time, call, etc. columns will always  be narrow, so the arguments and results column will be able to grow as needed.

stof reviewed (approved):
👍

fabpot commented:
Thank you @javiereguiluz.
